### PR TITLE
Added Savio Martin to the list

### PR DESCRIPTION
**Pull request by saviomartin**
**Added Savio Martin** to `data.json` **with image and issue number: 10**

Hope it makes sense.
Thank you

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -1,4 +1,10 @@
 [
+    {
+        "name": "Savio Martin",
+        "githubUsername": "saviomartin",
+        "imageUrl": "https://raw.githubusercontent.com/saviomartin/saviomartin/master/assets/saviomartin.png",
+        "issueNumber": 10
```
Comment (AllanRegush) [inline data.json:6]: Issue Number 10 is https://github.com/EddieJaoudeCommunity/awesome-github-profiles/issues/10
I see that you have raised an issue:
https://github.com/EddieJaoudeCommunity/awesome-github-profiles/issues/98
Shouldn't this be 98?

**Review by AllanRegush — COMMENTED** [review]
Looks Good :+1: Just a small adjustment

File: data.json (hunk 1 above)
Comment (saviomartin) [reply data.json]: **Oh Sorry**, I didn't think like that. I just took a number.
**I  have updated it.** 

Is that enough?
thankyou

**Comment by eddiejaoude** [comment]
Note: I think people usually append to the end of the `json` not too appear at the top - note to myself, we can fix this in the GitHub Action to sort by a value

**Pull request merged**